releasing/lagom.md

```diff
@@ -56,6 +56,10 @@ If there's been changes in the build or bumped versions of sbt plugins, you may
 
 ## Before You Release
 
+### Version bumps
+
+Check when was the last time scala-steward run on the repository and consider running it manually (it only runs mothly otherwise).
```

Comment:
```suggestion
Check when was the last time scala-steward run on the repository and consider running it manually (it only runs monthly otherwise).
```